docs/install.rst

```diff
@@ -268,8 +325,8 @@ Now you can login to the machine by ``ssh machine1``.
 Source: http://nerderati.com/2011/03/17/simplify-your-life-with-an-ssh-config-file/
 
 
-Setup GSISSH Access to a machine
----------------------------------
+Setup GSISSH Access to HPC resources
+------------------------------------
 
 Setting up GSISSH access to a machine is a bit more complicated. We have documented the steps to setup GSISSH on
```

Comment:
I would start the paragraph with: `Currently, GSISSH is unsupported but several HPC platforms still require it.`